salt/covid19/public.sls

```diff
@@ -0,0 +1,53 @@
+{% from 'lib.sls' import apache, create_user %}
+
+
+include:
+  - apache
+
+curl:
+  pkg.installed
+
```

Comment:
curl is part of the base installation. I think we can remove it here (otherwise, it probably belongs in the `core/` state files).